include/ppx/knob.h

```diff
@@ -124,7 +126,7 @@ class KnobSlider final
     static_assert(std::is_same_v<T, int>, "KnobSlider must be created with type: int");
 
     KnobSlider(const std::string& flagName, T defaultValue, T minValue, T maxValue)
-        : Knob(flagName), mValue(defaultValue), mDefaultValue(defaultValue), mMinValue(minValue), mMaxValue(maxValue)
+        : Knob(flagName, true), mValue(defaultValue), mDefaultValue(defaultValue), mMinValue(minValue), mMaxValue(maxValue)
```

Comment:
I'd expose the visible boolean in the constructor params (with default value to true)

Reply:
Fair point, fine with that.